modules/tinymce/src/themes/silver/test/ts/browser/editor/ToolbarPersistTest.ts

```diff
@@ -0,0 +1,57 @@
+import { Chain, Log, Pipeline } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { Editor as McEditor, UiChains } from '@ephox/mcagar';
+import { SugarElement, SugarBody, Insert, Focus, Remove, Visibility } from '@ephox/sugar';
+import { Fun } from '@ephox/katamari';
+import Theme from 'tinymce/themes/silver/Theme';
+import Editor from 'tinymce/core/api/Editor';
+
+UnitTest.asynctest('Toolbar persist test', (success, failure) => {
+  Theme();
+
+  const cWaitForVisible = UiChains.cWaitForState(Visibility.isVisible);
```

Comment:
Is it worthwhile to just add these to `UiChains`? Also, isn't this just `UiChains.cWaitForPopup`?

Reply:
Yeah that's right, `UiChains.cWaitForPopup` is the same. Using that instead.